src/app/scss/layout/_header.scss

```diff
@@ -0,0 +1,156 @@
+header.main-header {
+  display: flex;
+  justify-content: center;
+  @include mt(4);
+
+  .mesh-image {
+    display: none;
+  }
+
+  .bike-img {
+    display: none;
+  }
+  .header-info {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: -16px;
+    margin-bottom: -16px;
+
+    & > * {
+      @include my(4);
+    }
+  }
+  .main-heading {
+    font-size: 48px;
+    text-align: center;
+    @include mb(3)
+  }
+  .working-hours {
+    width: 90%;
+    .desktop {
+      display: none;
+    }
+    article {
+      max-width: 150px;
+
+      p {
+        font-size: 21px;
+      }
+    }
+    .first-row {
+      display: flex;
+      justify-content: space-between;
+      width: 100%;
+      margin: -16px;
+      & > * {
+        @include my(7);
+      }
+    }
+    .second-row {
+      width: 50%;
+    }
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  .sub {
+    font-family: Montserrat-SemiBold;
+    text-align: center;
+    @include mb(6);
+  }
+  .action-buttons {
+    display: flex;
+    width: 100%;
+    margin: -9px;
+
+    .white-button {
+      svg {
+        min-width: 26px;
+      }
+    }
+
+    & > * {
+      width: 50%;
+      display: flex;
+      align-items: center;
+      justify-content: center;
```

Comment:
`base-button` class should have `display: flex;` with `align-items: center;` as far as I remember.